packages/core/src/Services/logout.js

```diff
@@ -13,6 +13,7 @@ const doLogout = response => {
     removeCookies('affiliate_token', 'affiliate_tracking', 'onfido_token');
     localStorage.removeItem('closed_toast_notifications');
     localStorage.removeItem('is_wallet_migration_modal_closed');
+    localStorage.removeItem('active_wallet_loginid');
```

Comment:
so we hardcode this string here and there - probably it should be a constant somehwere (I know it used to be like that in legacy code, but generally its not best pattern)

Reply:
Agreed, dunno where to store this constant though